test/csuite/parse_opts/main.c

```diff
@@ -0,0 +1,315 @@
+/*-
```

Comment:
When I first read this my first thought was "this belongs in utility/parse_opts.c" thinking it was code to do the parsing, not test the parsing. Maybe call it `test/csuite/wt9937_parse_opts` since all the `wt<number>` tests are all tests? This is definitely a preference thing so ignore if you don't like that idea.

Reply:
Fixed.